internal/util/portrangeutil/util.go

```diff
@@ -0,0 +1,13 @@
+package portrangeutil
```

Comment:
Can you move functions in this package to `internal/util` unless there is some cyclic dependency issue?

Reply:
sure